### ISSUE TITLE
Scribite plugin add more options for inserting media/collections

### ISSUE THREAD

**Issue by espaan** [cmfcmf/MediaModule#25]
Hi,
when you insert for exampe a collection it would be nice to have more options, like 
for Collections:
 * insert as a plain link (as it is now)
 * insert as a single thumbnail with a link to the collection page
 * insert as a set of thumbnails (with a certain max number for instance) linking directly to the images and being able to arrow forward/backward
 * insert as a photo slider with the photos of that collection

For Media
 * insert small media with modal/popup to full image
 * insert plain media with a certain size (as it is now small/medium/large)
 * insert several media at once

**Comment by cmfcmf** [comment]
Yeah, the Scribite plugin is still a little rough. Thank you for your
suggestions!

Erik Spaan <notifications@github.com> schrieb am Di., 27. Okt. 2015 16:09:

**Comment by rallek** [comment]
I would like as well a better selection handling.
